public/index.html

```diff
@@ -5,11 +5,28 @@
     <link rel="icon" href="%PUBLIC_URL%/fcc-favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+    <!--Twitter cards information-->
     <meta
       name="description"
       content="This is a developer quiz made by freeCodeCamp"
     />
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@freeCodeCamp" />
+    <meta name="twitter:creator" content="@freeCodeCamp" />
+    <meta property="og:url" content="https://developerquiz.org" />
+    <meta property="og:title" content="Learn to Code RPG Quiz" />
+    <meta
+      property="og:description"
+      content="A website with over 700 questions to practice with."
```

Comment:
Let's change this to 1000 since we are close to hitting that

```suggestion
      content="A website with over 1000 questions to practice with."
```

Reply:
Woah. I had no idea.